### PR TITLE
Bibliography with similar entities

### PR DESCRIPTION
**Pull request by carcruz**
Disease page

### CODE REVIEW

**Comment by andrewhercules** [comment]
Great work @carcruz! 👍 

Below are a few requests for changes based on a review of the deploy preview. 

### Summary widget text

For the summary widget at the top of the page, please update to "n publications" 
Screenshot 2021-04-21 at 12 32 35

### Detail view description text

For the description of the section, please update to:

> "Scientific literature mentioning NLP-recognised entity **XXXXX** and other selected co-occurring entities. Source: [Open Targets](http://platform-docs.opentargets.org/bibliography)". 

Please replace **XXXXX** with the target/disease/drug name and please link "Open Targets" to http://platform-docs.opentargets.org/bibliography

### Data table

Please fix the bug where the "Show more rows" doesn't work (currently only allows 5).

Also, if possible, please use the Known Drugs loading state approach when requesting next batch of publications from Europe PMC. In the Known Drugs section, we load the first 100 records and when the component loads the next 100 records, it keeps the same page in view and the loading indicator is displayed near the pagination controls. 

### Tag category dropdown

If a user selects an option in the "Tag category" dropdown, it changes the results displayed in the table - it should only change the entity chips, leaving the results the same. When selecting a category, we only need to update the `similarEntities` query (e.g. [melanoma](https://api-beta-dot-open-targets-eu-dev.appspot.com/api/v4/graphql/browser?query=query%20melanomaBibliography%20%7B%0A%20%20disease%28efoId%3A%20%22EFO_0000756%22%29%20%7B%0A%20%20%20%20similarEntities%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20score%0A%20%20%20%20%20%20category%0A%20%20%20%20%7D%0A%20%20%20%20literatureOcurrences%20%7B%0A%20%20%20%20%20%20count%0A%20%20%20%20%20%20rows%20%7B%0A%20%20%20%20%20%20%20%20pmid%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)), passing the `(entityNames:["xxxxx"])` parameter (e.g. [melanoma with category == 'targets'](https://api-beta-dot-open-targets-eu-dev.appspot.com/api/v4/graphql/browser?query=query%20melanomaBibliography%20{%20disease(efoId:%20%22EFO_0000756%22)%20{%20similarEntities(entityNames:[%22target%22])%20{%20id%20score%20category%20}%20literatureOcurrences%20{%20count%20rows%20{%20pmid%20}%20}%20}%20})).  

### Hiding chips if no publications

After selecting multiple chips, there are no results returned if the `literatureOccurences.count == 0`. But due to the preset thresholds, there may be similar entities returned. 

If there are no publications available, please remove the chips from the view and display a message that the user can remove chips to see publications. 

For example, after selecting multiple chips on the [Wilson disease profile page](https://deploy-preview-324--platform-app.netlify.app/disease/Orphanet_905), this is what the user will see:

Screenshot 2021-04-21 at 12 01 00

Instead, could we please show them this view:

Screenshot 2021-04-21 at 12 58 47

**Comment by carcruz** [comment]
Thanks for the review @andrewhercules. All your comments were addressed on the last PR build

File: src/sections/target/Bibliography/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const definition = {
+  id: 'bibliography',
+  name: 'Bibliography',
+  shortName: 'B',
+  hasData: () => true,
```
Comment (LucaFumis) [inline src/sections/target/Bibliography/index.js:5]: Just double-checking that is the intended behaviour (we always have literature data)

**Review by LucaFumis — APPROVED** [review]
LGTM @carcruz (other than the question about the `hasData() => true`)
Perhaps if we use that "no data" also in other places in the future it could be extracted to the consts file, like the "N/A".

**Comment by carcruz** [comment]
Thanks @LucaFumis, the `hasData` validation was fixed in the last commit, can be tested with `/disease/EFO_0004311`

**Pull request merged**